Source/SwiftLintFramework/Rules/RuleConfigurations/FileHeaderConfiguration.swift

```diff
@@ -38,9 +38,9 @@ public struct FileHeaderConfiguration: RuleConfiguration, Equatable {
     private static let defaultRegex = regex("\\bCopyright\\b", options: [.caseInsensitive])
 
     public var consoleDescription: String {
-        return severityConfiguration.consoleDescription + ", required_string: \(requiredString)" +
-            ", required_pattern: \(requiredPattern), forbidden_string: \(forbiddenString)" +
-            ", forbidden_pattern: \(forbiddenPattern)"
+        return severityConfiguration.consoleDescription + ", required_string: \(String(describing: requiredString))" +
```

Comment:
Since any of these are optional, should we just replace them with `"None"` in the case that we're missing them?